agir/events/components/eventPage/EventLocationCard.js

```diff
@@ -30,6 +30,18 @@ const WithLinebreak = styled.span`
 const MapContainer = styled.div`
   margin: -1.5rem -1.5rem 1.5rem;
 
+  @media (max-width: ${style.collapse}px) {
+    margin: 24px 0 0 0;
+
+    * {
+      border-radius: 8px;
```

Comment:
On a une variable maintenant pour la nouvelle valeur de border-radius :
```suggestion
  border-radius: ${style.borderRadius};
```